lte/gateway/python/magma/kernsnoopd/BUILD.bazel

```diff
@@ -21,10 +21,7 @@ LTE_ROOT = "{}lte/gateway/python".format(MAGMA_ROOT)
 py_binary(
     name = "kernsnoopd",
     srcs = ["main.py"],
-    data = [
-        "ebpf/byte_count.bpf.c",
-        "ebpf/common.bpf.h",
-    ],
+    data = ["//lte/gateway/python/magma/kernsnoopd/ebpf:ebpf_data_files"],
```

Comment:
These might be duplicated somehow. I see them both in
```
└── var
    ├── opt
    │   └── magma
    │       ├── ebpf
    │       │   ├── ebpf_dl_handler.c
    │       │   ├── ebpf_manager.py
    │       │   ├── ebpf_ul_handler.c
    │       │   └── kernsnoopd
    │       │       ├── byte_count.bpf.c
    │       │       └── common.bpf.h
    │       └── envoy.yaml
```
and usr/local/lib/python3.8
```
│       │           │   ├── kernsnoopd
│       │           │   │   ├── ebpf
│       │           │   │   │   ├── byte_count.bpf.c
│       │           │   │   │   └── common.bpf.h
│       │           │   │   ├── handlers.py
│       │           │   │   ├── kernsnoopd
│       │           │   │   ├── main.py
│       │           │   │   ├── metrics.py
│       │           │   │   └── snooper.py
```

Reply:
addressed